### DISCUSSION TITLE
某些系统下通知栏的小图标变白

### DISCUSSION THREAD

**Discussion by Sasa-kid** [gkd-kit/gkd#783] [Announcements]
### 日志文件

https://f.gkd.li/17841438

### BUG描述(文字/截图/视频)

![Screenshot_20241121_151956](https://github.com/user-attachments/assets/cdcbdae1-e593-44f5-a0f5-c5ad26f69fdf)
![Screenshot_20241121_151955](https://github.com/user-attachments/assets/0dc05c2f-c5fd-49da-b379-f4fa64c8f437)
小图标变成白色的，打开通知栏又是正常的

### 期望行为(文字/截图/视频)

希望能修复，和旧手机的不一样

### 实际行为(文字/截图/视频)

新手机通过自带的数据迁移安装的，安装之后发现不对劲就升级为测试版，但还是一样的白色图标

**Comment by lisonge** [comment]
你好，应该是你的设备问题，我不知道怎么解决

<img src="https://github.com/user-attachments/assets/af3b057b-cb39-47bf-9caf-5197efc8ed83" width="50%" />

**Reply by lisonge** [reply]
可能是系统主题的问题，你切换到深色/浅色主题后再重新安装呢？

**Reply by Sasa-kid** [reply]
主题我没有改过，但在自带的几个主题里，我改过去然后重新下载安装也是一样的情况，手机是红魔10pro+，安卓版本是15，可能是系统的bug吧，解决不了也无所谓，功能一切正常

**Comment by chca2** [comment]
感觉是OS把图标进行了抠图，黑色背景白色内容的icon就会被“整合”成了纯白图标，因为看到其他彩色图标不会有这个问题。
因此看下开发者是否考虑对内容填充个其他颜色呢，或者反色处理，采用白色背景黑色主体。

**Reply by lisonge** [reply]
这个通知栏图标使用的并不是应用图标，而是根据 android 原生规范使用的另一个独立的纯白图标，它是透明背景
![image](https://github.com/user-attachments/assets/ca8e4046-9095-4e78-9392-1f11f7a5855f)

在原生 android 上就是按模拟器截图显示一个纯白/纯黑小图标，颜色会随通知栏文本颜色改变
![image](https://github.com/user-attachments/assets/89a4f092-f3e6-4cae-a4bc-f3be7be2a241)

在国产 android 如小米系统上就是强制使用 app 图标，app 图标是啥，这个图标就是啥

最后我不知道你的系统对通知这块魔改了什么，也不知道怎么改

**Reply by chca2** [reply]
没事，确实不影响使用。强迫症也可以去掉状态栏显示。
